themes/classic/templates/catalog/_partials/active_filters.tpl

```diff
@@ -1,6 +1,6 @@
 {if $activeFilters|count}
   <section class="active_filters">
-  <h1 class="h4 active-filter-title">{l s='Active filters' d='Shop.Theme'}</h1>
+  <h1 class="h6 active-filter-title">{l s='Active filters' d='Shop.Theme.Actions'}</h1>
```

Comment:
Wrong domain. It has been changed [here](https://github.com/PrestaShop/PrestaShop/pull/5844/files#diff-519de9779b6a0afc08f900719af03dc5R3) and I think you mis-rebased